positron/components/PositronAppRunner.jsm

```diff
@@ -53,14 +54,14 @@ PositronAppRunner.prototype = {
     return this._unboundLoadAndContinue.bind(this, aThenWhat.bind(this));
   },
 
-  _loadMainScript: function par_LoadMainScript(aScriptName) {
-    let mainScript = this._baseDir.clone();
-    mainScript.append(aScriptName);
+  _loadMainScript: function par_LoadMainScript(aScriptPath) {
+    let mainScriptPath = OS.Path.join(this._baseDir.path, aScriptPath);
+    let mainScriptURI = OS.Path.toFileURI(mainScriptPath);
     let requirer = {
       id: "resource:///modules/PositronAppRunner.jsm",
       exports: {},
     };
-    (new Require(requirer))(Services.io.newFileURI(mainScript).spec);
+    (new Require(requirer))(mainScriptURI);
```

Comment:
> And yes, we can also s/let/const/, although in this case we don't actually need to persist a reference to the module loader, so it can be even more simply:
> 
> (new ModuleLoader("browser")).require(requirer, mainScriptURI);

Fixed in conjunction with a merge from master.